### PR TITLE
Clarify slab decomposition and optional arguments

### PR DESCRIPTION
**Pull request by mathrack**

### CODE REVIEW

File: docs/source/pages/api_domain.rst [hunk 1]
```diff
@@ -156,17 +156,26 @@ The algorithm will choose the closest combination such as
   n\_proc=n\_col=\sqrt{nproc}
 
 In case the root is not exact the closest combitation to have :math:`n\_proc \approx n\_col` with 
-`n\_proc <  n\_col` is used. 
+`n\_proc <  n\_col` is used.
```
Comment (pbartholomew08) [inline docs/source/pages/api_domain.rst]: I think this (also in the original) is wrong - it should be p_row not n_proc, otherwise it doesn't make sense?

**Review by pbartholomew08 — CHANGES_REQUESTED** [review]

File: docs/source/pages/api_domain.rst (hunk 1 above)
Comment (mathrack) [reply docs/source/pages/api_domain.rst]: Probably yes. This is not part of the PR but I will include the fix in the PR

**Comment by mathrack** [comment]
Sorry, this math markdown stuff is driving me crazy. I am giving up on this.

**Review by pbartholomew08 — APPROVED** [review]

**Comment by pbartholomew08** [comment]
> Sorry, this math markdown stuff is driving me crazy. I am giving up on this.

Markdown sucks. I find it (can be) easiest to use the online editor and the "preview" button to see what it's becoming. But I also gave up and just used 'x' for '\times' ;)

**Pull request merged**